### DISCUSSION TITLE
Problem at lichesd

### DISCUSSION THREAD

**Discussion by apollonios2** [community/community#32536] [Other Feature Feedback, Questions, & Ideas] [answered]
Please put more time to wait at first move before cancel the game on lichess

**Comment by airtower-luna** [comment, marked as answer]
It looks like you're looking for help with a software hosted on GitHub, though I'm not sure which. In that case you should open an issue *in the repository for that software* so the developers see it. :slightly_smiling_face:

**Comment by jfullstackdev** [comment]
maybe it's this one 

https://github.com/lichess-org/lila

**Comment by PaquitoFIMO** [comment]
Sorry for my bad english, 
j'utilise GOOGLE TRANSLATE
VOIR IMAGE en fin de message
Look picture in end this message.


Hello,
For two days, or 3, it is impossible to download some pgns on tournaments, example here:

https://lichess.org/api/tournament/E3X0eqQ3/games

There is a network error:

Secure Connection Failed
An error occurred while connecting to lichess.org.
    The page you are trying to view cannot be displayed because the authenticity of the data received cannot be verified.
    Please contact the website owners to let them know about this issue.
And when downloading I get a "NETWORK FAILURES" error

Cordially

Cordialement
PaquitoFIMO sur lichess

MESSAGE ORIGINAL
Bonjour,
Depuis deux jours, ou 3, il est impossible de télécharger certains pgns sur les tournois, exemple ici : 

https://lichess.org/api/tournament/E3X0eqQ3/games

Il y a une erreur réseau : 

Échec de la connexion sécurisée
Une erreur est survenue pendant une connexion à lichess.org.
    La page que vous essayez de consulter ne peut pas être affichée car l’authenticité des données reçues ne peut être vérifiée.
    Veuillez contacter les propriétaires du site web pour les informer de ce problème.
Et lors du téléchargement, j'ai une erreur "ÉCHECS RÉSEAU"

Cordialement
PaquitoFIMO sur lichess

![2022_11_26_erreur_lichess](https://user-images.githubusercontent.com/119107474/204085550-c083ce2e-7348-425f-b167-bbb6c4c5a493.jpg)

**Reply by airtower-luna** [reply]
I'm afraid this is the wrong place for your report. This is the GitHub community, mostly for discussions around Git and GitHub. You should report the problem to the Lichess developers (make sure to mention it's an issue with the website): https://github.com/lichess-org/lila/issues

**Reply by PaquitoFIMO** [reply]
Sorry, and sorry for my bad english, and tank you vVERY much :)
